q2_demux/_summarize/_visualizer.py

```diff
@@ -55,50 +55,47 @@ def _link_sample_n_to_file(files, counts, subsample_ns):
 
 def _subsample_paired(fastq_map):
     qual_sample = collections.defaultdict(list)
-    seq_len = None
+    min_seq_len = {}
+    min_seq_len['forward'] = float('inf')
+    min_seq_len['reverse'] = float('inf')
+    max_seq_len = 0
```

Comment:
It isn't clear to me why we are tracking `max_seq_len` --- perhaps this is leftover code from a previous iteration of this PR? If so, can you please remove all references, if not, let's chat in person.